modules/beetmover_scriptworker/manifests/settings.pp

```diff
@@ -148,24 +150,21 @@
             verify_cot_signature                    => true,
             cot_product                             => 'thunderbird',
         },
```

Comment:
Should we define the `mobile-staging` too since we're here and we have new credentials?

Reply:
Let's do it in a follow up. We don't have all creds ready yet